libr/anal/bb.c

```diff
@@ -172,7 +170,10 @@ R_API void r_anal_bb_set_offset(RAnalBlock *bb, int i, ut16 v) {
 	if (i > 0) {
 		if (i >= bb->n_op_pos) {
 			bb->n_op_pos = i * 2;
-			bb->op_pos = realloc (bb->op_pos, bb->n_op_pos * sizeof (*bb->op_pos));
+			ut16 *tmp_op_pos = realloc (bb->op_pos, bb->n_op_pos * sizeof (*bb->op_pos));
+			if (!tmp_op_pos) return;
+			bb->op_pos = tmp_op_pos;
+
```

Comment:
no need for this line. Also, more important, since you used a temporary variable to do the realloc, you should also use a temporary var for n_op_pos, so that you set `bb->n_op_pos = i * 2` only when you are sure that the realloc went ok.